app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: :User, foreign_key: :author_id
+  belongs_to :post
+  after_save :update_post_likes_counter
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- **[OPTIONAL]** I would suggest you make this method private as it serves as a callback for this model and to restrict its access to this class.